testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/IdentityProviderTest.java

```diff
@@ -458,6 +458,7 @@ public void failUpdateInvalidUrl() throws Exception {
 
             try (Response response = realm.identityProviders().create(representation)) {
                 assertEquals(Response.Status.CREATED.getStatusCode(), response.getStatus());
+                getCleanup().addIdentityProviderAlias(representation.getAlias());
```

Comment:
You could perhaps leverage [`Creator.create`](https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/updaters/Creator.java#L115) in the outer `try` instead.

Reply:
I used the private `create` method in the same class as it's what all the other methods are doing. If you prefer the `Creator.create` let me know.

Reply:
I waiter the test to finish to check that the `IdentityProviderTest` test didn't fail in the initial run.